datacapture/src/main/java/com/google/android/fhir/datacapture/DataCaptureConfig.kt

```diff
@@ -42,10 +45,22 @@ object DataCaptureConfig {
    * needed by [StructureMap]s used by the client app.
    */
   var npmPackage: NpmPackage? = null
+) {
 
   internal val simpleWorkerContext: SimpleWorkerContext by lazy {
     if (npmPackage == null) SimpleWorkerContext() else SimpleWorkerContext.fromPackage(npmPackage)
   }
+
+  /**
+   * A class that can provide the [DataCaptureConfig] for the Structured Data Capture Library. To do
+   * this:
+   * * Implement the {@link DataCaptureConfig.Provider} interface on your [Application] class. You
```

Comment:
sorry, let's just inline the sentence.

```suggestion
   * this, implement the {@link DataCaptureConfig.Provider} interface on your [Application] class. You
```

Reply:
Done